### ISSUE TITLE
communicate collection_id in the portal

### ISSUE THREAD

**Issue by fedorov** [ImagingDataCommons/IDC-WebApp#1354]
`collection_id` is needed for filtering or downloading data, and currently there is no easy and intuitive way to connect the names of the collections shown in the portal with the collection id. We need to fix this.

One idea is to allow user to copy `collection_id` into clipboard, while showing collection name in the interface. I am open to alternative suggestions, but we need to fix this.

**Comment by G-White-ISB** [comment]
You could have a copy icon in the collection table that copies the collection_id rather than the collection name

**Comment by fedorov** [comment]
Sounds good - let's add this along with the case ID in #1350?

**Comment by s-paquette** [comment]
Which version of the ID did we want to use? There's a few different ones; IIRC `collection_id` is our 'webapp ID' i.e. it's one we constructed for use with the WebApp.

**Comment by fedorov** [comment]
The one that we have in the `collection_id` column of our `dicom_all` table.

**Comment by s-paquette** [comment]
Got it.

**Comment by fedorov** [comment]
Agreed to add a "copy to clipboard" button and the tooltip will show "Copy collection_id to the clipboard".

<img width="420" alt="image" src="https://github.com/ImagingDataCommons/IDC-WebApp/assets/313942/4ca823dd-abb8-4a5e-b0aa-d329801aebbc">

Instead of showing collection information in the tooltip as done right now, use the same approach as we do in the left panel to show info button

<img width="318" alt="image" src="https://github.com/ImagingDataCommons/IDC-WebApp/assets/313942/e30ca12d-df8f-41f4-8aa8-26323a6eb4c0">

**Comment by G-White-ISB** [comment]
Info buttons and clipboards?? I'm not a great judge of aesthetics, but even for me that's a little ugly.
